_redirects

```diff
@@ -39,3 +39,6 @@ https://developer.aiven.io/* https://docs.aiven.io/:splat 301!
 # Renamed/deleted files
 /docs/products/flink/howto/real-time-alerting-solution-cli.html       /docs/products/flink/howto/real-time-alerting-solution.html
 /docs/platform/howto/disable-user-2fa.html                            /docs/platform/howto/user-2fa.html
+/docs/products/mysql/howto/max-number-of-connections.html             /docs/products/mysql/concepts/max-number-of-connections.html
+/docs/products/postgresql/reference/plan-iops.html                    /docs/products/postgresql/reference/resource-capability.html
+/docs/tools/terraform/upgrade-provider-v1-v2                          /docs/tools/terraform/howto/upgrade-provider-v1-v2
```

Comment:
This redirect doesn't seems to work - notice in [preview](https://deploy-preview-1322--developer-aiven-io-preview.netlify.app/docs/tools/terraform/upgrade-provider-v1-v2). Will test something out in this PR